src/domain/synthetics/tokens/useTokenAllowanceData.ts

```diff
@@ -51,6 +58,43 @@ export function useTokensAllowanceData(
       }, {} as TokensAllowanceData),
   });
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const stableTokenAddresses = useMemo(() => tokenAddresses as Address[], [tokenAddresses.join("-")]);
```

Comment:
"stable" means stable coin throughout the project which is confusing, let's use serialized prefix instead?

Reply:
trying to understand why is it serialized, it just returns tokenAddresses array?